test/test_jit.py

```diff
@@ -14513,15 +14513,15 @@ def forward(self):
             @torch.jit.export
             def __setstate__(self, state):
                 self._packed_weight.set_(
-                    torch.ops.quantized.linear_prepack(state))
+                    torch.ops.quantized.linear_prepack(state, None))
```

Comment:
If we have `c10::optional<Tensor> bias`, we don't need to add None here and provides the backward compatibility.